scripts/run_e2e_tests.py

```diff
@@ -37,6 +38,8 @@
 OPPIA_SERVER_PORT = 8181
 PROTRACTOR_BIN_PATH = os.path.join(
     common.NODE_MODULES_PATH, 'protractor', 'bin', 'protractor')
+PORTSERVER_SOCKET_PATH = 'portserver.socket'
```

Comment:
Maybe give some more explanation of this (and/or a clearer name) since it doesn't actually look like a path (e.g. I have no idea what the root dir is, etc.).

Reply:
This is a path relative to the current working directory. It will create a file at `./portserver.socket`. I'll add a comment to explain this

Reply:
done in 97bba1a74

Reply:
Ah -- can you maybe rename to change _PATH to _FILEPATH, and also make it more robust by having some sort of os.path.join(os.getcwd(), ...) here? Maybe also update the description above to say "where the portserver socket file will be created".

(This will make clearer that it points to a file. At first I thought it was some kind of internal URL.)